installer/tue-install-impl.bash

```diff
@@ -866,20 +866,45 @@ function _tue-install-pip-now
         tue-install-debug "Already pip${pv}>=$desired_pip_version\n"
     fi
 
+    local pips_to_check=""
     local pips_to_install=""
     local git_pips_to_install=""
     # shellcheck disable=SC2048
     for pkg in $*
     do
-        local installed_version
         if [[ "$pkg" == "git+"* ]]
         then
             git_pips_to_install="$git_pips_to_install $pkg"
-        elif ! installed_version=$(python"${pv}" "$TUE_INSTALL_SCRIPTS_DIR"/check-pip-pkg-installed-version.py "$pkg")
+        else
+            pips_to_check="$pips_to_check $pkg"
+        fi
+    done
+
+    read -r -a pips_to_check <<< "$pips_to_check"
+    local installed_versions
+    installed_versions=$(python"${pv}" "$TUE_INSTALL_SCRIPTS_DIR"/check-pip-pkg-installed-version.py "${pips_to_check[@]}")
```

Comment:
There might not be any pips to check (all git+ notation). Then this command fails because of no input.